### PR TITLE
New benchmark to measure performance of animations after removing a BackdropFilter.

### PR DESCRIPTION
**Pull request by flar**

### CODE REVIEW

File: dev/devicelab/bin/tasks/post_backdrop_filter_perf__timeline_summary.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter_devicelab/tasks/perf_tests.dart';
+import 'package:flutter_devicelab/framework/adb.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+Future<void> main() async {
```
Comment (liyuqian) [inline dev/devicelab/bin/tasks/post_backdrop_filter_perf__timeline_summary.dart]: Considering that our devicelab's resources are currently very limited, maybe we don't have to add the Android test until we find good use of it. I think with or without your optimization, the performance on Android should be similar?

**Review by liyuqian — COMMENTED** [review]
The iOS one looks good. Have a little question on why we need the Android one.

File: dev/devicelab/bin/tasks/post_backdrop_filter_perf__timeline_summary.dart (hunk 1 above)
Comment (flar) [reply dev/devicelab/bin/tasks/post_backdrop_filter_perf__timeline_summary.dart]: That makes sense, I'll also remove the entry from the manifest.

**Review by liyuqian — APPROVED** [review]
LGTM

**Pull request merged**